changelog-generator/content/changelogs/20231103_users_service-1-3-0.md

```diff
@@ -0,0 +1,19 @@
+---
+service: users-service
+version: 1.3.0
+date: 2023-11-03
+---
+
+**🎁 Features**
+* Disable the total count in the listing query
+  * Added a `skip_count()` RQL operator. Would instruct the listing functions not to execute/return the total count. 
```

Comment:
I think I said this to Aäron when he copy pasted this a while back too...

The first line can go 🤔

Reply:
This is part of the "real" / "official" release notes though. But fine for me to remove